common/src/main/java/oracle/kubernetes/common/logging/MessageKeys.java

```diff
@@ -145,6 +145,7 @@ public class MessageKeys {
   public static final String LOG_WAITING_COUNT = "WLSKO-0193";
   public static final String INTERNAL_IDENTITY_INITIALIZATION_FAILED = "WLSKO-0194";
   public static final String DOMAIN_FATAL_ERROR = "WLSKO-0195";
+  public static final String INTROSPECTOR_FATAL_ERROR = "WLSKO-0196";
```

Comment:
Something is off here because you have "WLSKO-0196" duplicated and Operator.properties is using "WLSKO-0195" for the new message.

Reply:
The new retry strategy requires different messages, and I was trying to reuse the numbers. Maybe that's an error.

Reply:
The String value, "WLSKO-0196", is the lookup key in to Operator.properties. If these two use cases need to generate the same message then this could work, but in that case I think it would clearer to use the same constant in MessageKeys.java.